src/service/platform-impl.js

```diff
@@ -158,6 +158,20 @@ export class Platform {
     version = version[1].replace(/_/g, '.');
     return version;
   }
+
+  /**
+   * Check whether current major IOS version is after specific version number.
+   * @param {number} version
+   * @return (boolean}
+   */
+  afterIosMajorVersion(version) {
```

Comment:
There are a lot fewer version positions than arithmetic operators :) So, I'd rather have `getIosMajorVersion()`, `getIosMinorVersion()` then a set of `before` and `after`... Instead, it'd be better to have a utility function that can be used for any version string. E.g.:
```
compareVersion(versionString, major, opt_minor, opt_evenMinorer, opt_etc) {
  const components = versionString.split('.').map(s => Number(s));
  if (components == ...) {
    return 0;
  }
  if (components < ...) {
    return -1;
  }
  return 1;
}
```

This will be the most reusable thing.